client/transactions/list/index.js

```diff
@@ -289,6 +296,24 @@ export const TransactionsList = ( props ) => {
 				'Search by order number, customer name, or billing email',
 				'woocommerce-payments'
 		  );
+
+	const title = props.depositId
+		? __( 'Deposit transactions', 'woocommerce-payments' )
+		: __( 'Transactions', 'woocommerce-payments' );
+
+	const onDownload = () => {
+		const { page, path, ...params } = getQuery();
+
+		downloadCSVFile(
+			generateCSVFileName( title, params ),
+			generateCSVDataFromTable( columnsToDisplay, rows )
+		);
+
+		window.wcTracks.recordEvent( 'wcpay_transactions_download', {
+			rows: transactionsSummary.count,
```

Comment:
I like that we're sending the total amount of transactions with this event. What do you think of also sending the number of rows in the exported `csv` so we can have some info on whether merchants opt to use a higher page size for exporting data?

Reply:
I think we should probably rename this event prop, as "rows" doesn't make it very clear that this is the number of transactions for that merchant.

Reply:
Agree to show also the exported rows in csv. What do you think of using `visible_transactions` and `total_transactions` ? Keep in mind that the total could be lower than all of the merchant transactions count because of filters.

Reply:
> Keep in mind that the total could be lower than all of the merchant transactions count because of filters.

Oh yeah, thanks for reminding me of this. I hadn't considered it in the naming suggestion.

> What do you think of using `visible_transactions` and `total_transactions`?

Looks good, but minor nitpick, do you think `exported_transactions` would be clearer?

Also, do you think we should inform, somehow, that `total_transactions` is the number of transactions that match the current filters? Or should we just clarify that on the event details page?

Reply:
Mmm, maybe we can use something like `exported_transactions` and `results_transactions` or something similar?

And yes, I think we should also add more clarification in event details page.

Reply:
Awesome, both look great! IMO we could also stick with `total_transactions`, if we add a clarification. But I'm fine either way and will defer it to you 😄